datamade/just-spaces
# Fix missing submit button on user signup page

## Overview

Closes #182. Adds back a missing submit button on the user signup form.

## Testing Instructions

 * Log out 
* Click "Create new account" in the sidebar
* Create a new account and make sure you can log in with it!
